core/src/commonMain/kotlin/io/github/aag/core/data/network/models/LessonDTO.kt

```diff
@@ -50,3 +50,26 @@ fun LessonDTO.toLecture(): Lesson =
         githubRepoUrl = githubRepoUrl,
         telegramChannel = telegramChannel
     )
+
+fun fromLesson(lesson: Lesson): LessonDTO =
```

Comment:
Can be extension function

Reply:
You mean:        
`fun Lesson.toLessonDTO(): LessonDTO`?         
then it'd look like the `domain` layer knows about `data` layer while it should not.